armi/bookkeeping/mainInterface.py

```diff
@@ -28,7 +28,7 @@
 from armi import utils
 from armi.utils import pathTools
 from armi import operators
-from armi.localization.exceptions import InputError
+from armi.utils.customExceptions import InputError
```

Comment:
Good instincts

Reply:
Just a head's up: those two plugins are used in 8 different plugins.  Do you still want me to delete them? 

(No opinion, just sharing information.)